server/controllers/messageController.js

```diff
@@ -1,11 +1,151 @@
 const db = require('../../database/postgres');
 
 /**
- * @route /api/message
+ * @route /api/messages?user1=?&user2=?
  * @method GET
- * @desc get a message
+ * @desc get all messages from user1 and user2
  */
+const getMessage = async (req, res) => {
+  const user1 = req.query.user1;
+  const user2 = req.query.user2;
+  const messages = await db.query(
+    `SELECT * FROM direct_messages WHERE sender_id=${user1} AND receiver_id=${user2} OR sender_id=${user2} AND receiver_id=${user1};`
+  );
+  if (messages) {
+    res.status(200).send(messages.rows);
+  } else {
+    res.send({ Error: 'some error' });
```

Comment:
when catching errors for the query, if you are using "await" you may want to wrap it into a try catch block. 
samples : https://node-postgres.com/features/queries

Reply:
and also for other await query places!